serving/samples/helloworld-nodejs/Dockerfile

```diff
@@ -8,7 +8,7 @@ WORKDIR /usr/src/app
 # Copy application dependency manifests to the container image.
 # A wildcard is used to ensure both package.json AND package-lock.json are copied.
 # Copying this separately prevents re-running npm install on every code change.
-COPY package*.json ./
+COPY package.json package*.json ./
```

Comment:
If we only have two files here, how about

```
COPY package.json package-lock.json ./
```

Reply:
I prefer to keep Dockerfiles building even if package-lock.json is deleted from an ease of testing/updating standpoint.

Reply:
make sense, thanks @grayside 

/lgtm